wger/manager/views/workout.py

```diff
@@ -20,12 +20,16 @@
 
 from django.shortcuts import render, get_object_or_404
 from django.http import HttpResponseRedirect, HttpResponseForbidden
+from django.http import HttpResponse
 from django.template.context_processors import csrf
 from django.core.urlresolvers import reverse, reverse_lazy
 from django.utils.translation import ugettext_lazy, ugettext as _
 from django.contrib.auth.mixins import PermissionRequiredMixin, LoginRequiredMixin
 from django.contrib.auth.decorators import login_required
 from django.views.generic import DeleteView, UpdateView
+# We will be able to create a ModelResource class
+# to describe how we export resource
+from import_export import resources
```

Comment:
It has been used the class that gets the items in the workout model as Json. Here: `class ExportWorkoutsResource(resources.ModelResource)`